File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Java/src/main/java/gameCatan/simulationCatan.java
package gameCatan;

import gameCatan.factories.ComputerFactory;
import gameCatan.factories.HumanFactory;
import gameCatan.factories.MapFactory;
import gameCatan.models.Computer;
import gameCatan.models.Human;
import gameCatan.models.Map;

import static gameCatan.utils.funciones.simulationCatan;

public class simulationCatan {
    public static void main(String[] args) throws InterruptedException {

        // Inicialización de nuestros objetos
        Map map = MapFactory.create();
        Human playerHumano = HumanFactory.create();
        Computer playerComputer = ComputerFactory.create();
        // Iniciamos simulación
        simulationCatan(map, playerHumano, playerComputer);
    }

}

File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Kotlin/gameCatan/simulationCatan.kt
package gameCatan

import gameCatan.factories.ComputerFactory
import gameCatan.factories.HumanFactory
import gameCatan.factories.MapFactory
import gameCatan.utils.funciones

fun main(args: Array<String>) {

    // Inicialización de nuestros objetos
    val map = MapFactory.create()
    val playerHumano = HumanFactory.create()
    val playerComputer = ComputerFactory.create()
    // Iniciamos simulación
    funciones.simulationCatan(map, playerHumano, playerComputer)
}